RELEASE_NOTES.md

```diff
@@ -199,6 +199,12 @@ v1.0.0-alpha.X
 
 ### Bug fixes
 
+- Python objects that inherit from a C++ base class, and are held by a
+  C++ object as a C++ base class pointer, will no longer be garbage
+  collected prematurely.
```

Comment:
[fix]
```suggestion
  C++ object as a C++ base class pointer, will no longer be deleted
  prematurely.
```

Reply:
Reworded in abc9de3d, though I went with "destroyed" rather than "deleted", if that's OK?